### ISSUE TITLE
Is piper optimized to run on a GPU? Would like to know before buying one

### ISSUE THREAD

**Issue by beneadie** [rhasspy/piper#443]
I am trying to build an API that uses piper and can deal with long text. I dont think it is viable to deploy it on a normal server with CPUs because it runs at 100% capacity and for long text can go on for a few minutes per request. I am now looking into the possibility of using a GPU server to deploy it. I am extremely new to GPUs and have never worked with one before. I know piper has some GPU functionality but cannot tell if this is only for the training stage and not for running it. I'd really appreciate some insight from anyone who has tried running it on a GPU before just so I know it will work before I invest in one. Thanks in advance 🙏

**Comment by synesthesiam** [comment]
Yes, Piper can definitely run on a GPU! I have only ever tested it on NVIDIA GPUs, though. Even an older GPU like a 1060 6GB can help quite a bit.

**Comment by beneadie** [comment]
Thanks so much! Any chance you could give me a rough idea of how much faster it makes it? I am super new to GPUs so sorry for these basic questions
